src/Dialect/ONNX/ONNXOps.cpp

```diff
@@ -3424,6 +3449,37 @@ LogicalResult ONNXDropoutOp::inferShapes(
 // OneHotEncoder
 //===----------------------------------------------------------------------===//
 
+static LogicalResult verify(ONNXOneHotEncoderOp op){
+  ONNXOneHotEncoderOpAdaptor operandAdaptor =
+      ONNXOneHotEncoderOpAdaptor(op);
+
+  // get operands
+  auto input = operandAdaptor.X();
+  if(!hasShapeAndRank(input)){
```

Comment:
Remove unnecessary `{}`. This is not done consistently, but we have decided to follow the LLVM style/coding guidelines which states:

https://llvm.org/docs/CodingStandards.html#don-t-use-braces-on-simple-single-statement-bodies-of-if-else-loop-statements